content/docs/09-registries-and-packs/1-adding-a-custom-registry.md

```diff
@@ -216,70 +203,124 @@ Registry servers configured in HTTP mode require the `--insecure` CLI flag when
 
-Once the deployment of the pack registry server is complete, configure it with the console as follows:
+  ```bash
+  curl -v [REGISTRY_SERVER]/health
+  curl -v -u [USERNAME] [REGISTRY_SERVER]/v1/_catalog
+  ```
+
+</Tabs.TabPane>
+
+</Tabs> 
 
-1. As a Tenant Administrator, navigate to **Admin Settings** > **Registries** > **Pack Registries**.
 
 
-2. Click on **Add New Pack Registry** and provide the pack registry name, endpoint, and user credentials.
+# Configure a Custom Pack Registry in Palette
 
+<<<<<<< HEAD
+Once you deploy the pack registry server, use the following steps to configure the pack registry server in Palette.
+<br />
+
+1. Log in to Palette, and switch to the tenant admin view. 
+
+=======
+# Configure a Custom Pack Registry in Palette
+
+Once you deploy the pack registry server, use the following steps to configure the pack registry server in Palette.
+<br />
 
-3. Click on **Confirm** once the details are filled.
+1. Log in to Palette, and switch to the Tenant admin view. 
 
-# Upload the CA Certificate to Palette
 
-In order to establish a trusted secure connection between Palette and your registry
-you will need to upload your certificate to the console.
+2. Navigate to the **Tenant Settings** > **Registries** > **Pack Registries** section.
+
+
+3. Click on the **Add New Pack Registry**. Palette will open a pop-up window asking for the fields to configure a pack registry server, as highlighted in the screenshot below. 
+
+  ![A screenshot highlighting the fields to configure a custom pack registry. ](/registries-and-packs_adding-a-custom-registry-tls_certificate.png)
+
+
+4. Provide the custom pack registry server name, endpoint, and user credentials in the pop-up window. Ensure to use an "https://" prefix in the pack registry server endpoint. 
+
+
+5. If you want Palette to establish a secure and encrypted HTTPS connection with your pack registry server, upload your TLS certificate issued from a Certificate Authority (CA) in the **TLS Configuration** section. Palette will perform the TLS verification to affirm the certificate's authenticity before establishing a communication channel. 
+>>>>>>> 5f11c0ce (docs: resolve conflict)
+
+2. Navigate to the **Tenant Settings** > **Registries** > **Pack Registries** section.
+
+<<<<<<< HEAD
+=======
+6. Select the **Insecure Skip TLS Verify** checkbox if you do not want an HTTPS connection between Palette and your pack registry server.  
+>>>>>>> 5f11c0ce (docs: resolve conflict)
+
+3. Click on the **Add New Pack Registry**. Palette will open a pop-up window asking for the fields to configure a pack registry server, as highlighted in the screenshot below. 
+
+<<<<<<< HEAD
+  ![A screenshot highlighting the fields to configure a custom pack registry. ](/registries-and-packs_adding-a-custom-registry-tls_certificate.png)
+=======
+7. Click the **Confirm** button to finish configuring the custom pack registry server.
+>>>>>>> 5f11c0ce (docs: resolve conflict)
```

Comment:
I'll defer to your to determine what is correct and what from the merge conflict needs to be removed.